features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/BaseWireframeMapper.kt

```diff
@@ -103,10 +103,12 @@ abstract class BaseWireframeMapper<T : View, S : MobileSegment.Wireframe>(
      */
     @MainThread
     protected fun handleBitmap(
+        view: View,
```

Comment:
maybe rather pass `Context` than a `View`? It seems more reasonable. Or `View` will be needed at some point in future?